src/components/scenes/MigrateWalletCalculateFeeScene.tsx

```diff
@@ -171,19 +167,19 @@ const MigrateWalletCalculateFeeComponent = (props: Props) => {
             const maxSpendInfo = { ...spendInfo, spendTargets: [{ publicAddress, nativeAmount: maxAmount }] }
             const edgeTransaction = await wallet.makeSpend(maxSpendInfo)
             const txFee = edgeTransaction.parentNetworkFee ?? edgeTransaction.networkFee
-            bundlesFeeTotals[asset.key] = txFee
+            bundlesFeeTotals.set(asset.key, txFee)
             feeTotal = add(feeTotal, txFee)
 
             // While imperfect, sanity check that the total fee spent so far to send tokens + fee to send mainnet currency is under the total mainnet balance
             if (i === bundle.length - 1 && lt(wallet.balances[asset.currencyCode], feeTotal)) {
               throw new InsufficientFundsError({ currencyCode: asset.currencyCode, networkFee: feeTotal })
             }
           } catch (e: any) {
-            for (const key of Object.keys(bundlesFeeTotals)) {
+            for (const key of [...bundlesFeeTotals.keys()]) {
```

Comment:
You don't need the `[...keys()]` bit, since `keys()` already returns an iterator that is compatible with `for` loops.

Reply:
Ah it's a `for..in` loop that it works. Didn't work for `for..of` and I've conditioned myself away from `for..in` so I just accepted it and did the ugly thing.